src/openforms/js/components/admin/forms/ReactSelect.js

```diff
@@ -31,19 +31,43 @@ const styles = {
   }),
 };
 
+/**
+ * A select dropdown backed by react-select for regular usage.
+ *
+ * Any additional props are forwarded to the underlying ReactSelect component.
+ */
+const SelectWithoutFormik = ({name, options, value, className, onChange, ...props}) => {
+  return (
+    <ReactSelect
+      inputId={`id_${name}`}
+      name={name}
+      className={`admin-react-select ${className || ''}`}
+      classNamePrefix="admin-react-select"
+      styles={styles}
+      menuPlacement="auto"
+      options={options}
+      value={options.find(opt => opt.value === value) || null}
+      onChange={selectedOption => {
+        onChange(selectedOption === null ? undefined : selectedOption.value);
+      }}
+      {...props}
+    />
+  );
+};
+
 /**
  * A select dropdown backed by react-select for Formik forms.
  *
- * Any additional props are forwarded to the underlyng ReactSelect component.
+ * Any additional props are forwarded to the underlying ReactSelect component.
  */
-const Select = ({name, options, ...props}) => {
+const SelectWithFormik = ({name, options, className, ...props}) => {
   const [fieldProps, , fieldHelpers] = useField(name);
   const {value} = fieldProps;
   const {setValue} = fieldHelpers;
   return (
     <ReactSelect
       inputId={`id_${name}`}
-      className="admin-react-select"
+      className={`admin-react-select ${className || ''}`}
```

Comment:
preferably use `import classNames from 'classnames';` for dynamic class name building. In the typescript projects, this is the `clsx` library instead :)

Reply:
I've added it :)